ui/src/main/res/layout/fragment_xltunai_status.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why is it just a linear layout? 
Does this used anywhere? If not please remove it.